model/configuration.py

```diff
@@ -265,6 +265,13 @@ class ExternalIntegration(Base, HasFullTableCache):
     USERNAME = u"username"
     PASSWORD = u"password"
 
+    # If the request should use a custom headers, put it here.
+    CUSTOM_ACCEPT_HEADER = u"custom_accept_header"
+
+    # If want to use an identifier different from <id>, use this config.
+    CUSTOM_IDENTIFIER = u"custom_identifier"
```

Comment:
Calling this CUSTOM_IDENTIFIER makes it sound like the configuration value should be _one specific identifier_. I recommend calling it something like PRIMARY_IDENTIFIER_SOURCE to avoid confusion -- this is the place you look for the identifier, not the identifier itself.